src/stores/petProfileStore.ts

```
@@ -0,0 +1,51 @@
+          },
+        })),
+      resetPetProfile: () => set({ petProfile: defaultPetProfile }),
+    }),
+    {
+      name: 'pet-profile-storage',
+    },
+  ),
+)
+
+export const defaultPetProfile: PetProfile = {
+  selectedBreed: '...' as any, // BREED_VALUES 중 하나로 기본값 지정
```

Biome (2.1.2)

[error] Shouldn't redeclare 'defaultPetProfile'. Consider to delete it or rename it.

'defaultPetProfile' is defined here:


(lint/suspicious/noRedeclare)